legal/association_statute.md

```diff
@@ -0,0 +1,224 @@
+Rozdział I: Podstawowe informacje
+
+§1 Stowarzyszenie działające na podstawie niniejszego Statutu nosi nazwę Stowarzyszenie Code for Poznań. Stowarzyszenie może używać nazwy skróconej Code for Poznań.
+
+§2 Terenem działalności Stowarzyszenia jest Rzeczpospolita Polska, przy czym w zakresie niezbędnym dla właściwego realizowania celów może ono prowadzić działalność także poza granicami Rzeczpospolitej Polskiej.
+
+§3 Siedzibą Stowarzyszenia jest miasto Poznań.
+
+Rozdział II: Cele
+
+§4 Celami Stowarzyszenia są:
+1. tworzenie nowych rozwiązań technologicznych oraz wsparcie w ich rozpowszechnianiu i wdrażaniu;
+2. edukowanie w zakresie zarządzania, tworzenia i wdrażania projektów informatycznych; 
+3. promowanie idei wytwarzania otwartego oprogramowania na rzecz społeczności lokalnych i organizacji, odpowiadających na realne problemy społeczne, wspierających aktywność lokalną oraz promujących odpowiedzialność za dobro wspólne;
+4. promocja i organizacja wolontariatu oraz działalności charytatywnej;
+5. działalność wspomagająca rozwój wspólnot i społeczności lokalnych;
+6. działalność na rzecz organizacji pozarządowych.
+
+Rozdział III: Realizacja celów
+
+§5 Stowarzyszenie realizuje swoje cele poprzez tworzenie innowacyjnego, skalowalnego i potrzebnego społecznie oprogramowania we współpracy z partnerami, odpowiedzialnymi przede wszystkim za analizę oraz definiowanie problemów i potrzeb, które mogą zostać rozwiązane przy pomocy nowoczesnych rozwiązań technologicznych. Wiąże się to z gromadzeniem i przetwarzaniem danych dla realizacji celów Stowarzyszenia.
```

Comment:
Chcemy realizować wyłącznie dewelopment (jak software-house) bez utrzymania? Jeżeli utrzymanie, możemy myśleć o partnerach technologicznych, zwłaszcza w zakresie infrastruktury, ale może także w przyszłości – techsoup – oprogramowania, zwłaszcza że niekiedy wdrożenie (niżej pytam co znaczy "dostarczyć") będzie wymagało pozyskanie oprogramowania na licencje (przykładowo projekt wybrał bazę danych Oracle, chociaż reszta została napisana przez Stowarzyszenie).

Reply:
To słowo "dostarcza", o które pytałeś w komentarzu poniżej chyba tworzy nam właśnie takie możliwości, a przynajmniej będzie pod co je podpiąć.